FlightPatternDetection/Models/HoldingPattern.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Microsoft.EntityFrameworkCore;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+namespace FlightPatternDetection.Models;
+
+[Table("Holding_Pattern")]
+public class HoldingPattern
+{
+    [Key, ForeignKey(nameof(Flight.Flight_Id))]
+    public int Flight_Id { get; set; }
+
+    public string Fixpoint { get; set; }
+
+    public int Laps { get; set; }
+
+    public enum Direction
+    {
+        Right,
+        Left
+    }
```

Comment:
This is a definition of a new enum. Not a reference to it. 
You should move the selected to another file and then add `public Direction Direction {get; set;}` here instead.

Reply:
An Enum class has been created instead